server_environment/__init__.py

```diff
@@ -17,4 +17,9 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
+from . import models
+# Add an alias to access to the 'serv_config' module as it is shadowed
+# the next line by an import of a variable with the same name.
+# We can't change the import of serv_config for backward compatibility.
```

Comment:
Add a comment to fix it on Odoo 12 migration?

Reply:
done